### ISSUE TITLE
web application not running

### ISSUE THREAD

**Issue by Tajjar** [Graylog2/graylog2-images#193]
I tried installing graylog using vagrant on a test sever and when I request the web interface I get the permanent error "graylog is restarting". I tried to check the services as a start and it shows the following:

run: elasticsearch: (pid 7907) 9s; run: log: (pid 1155) 906s
run: etcd: (pid 7930) 8s; run: log: (pid 1168) 906s
down: graylog-server: 0s, normally up, want up; run: log: (pid 1153) 906s
run: mongodb: (pid 7957) 7s; run: log: (pid 1156) 906s
run: nginx: (pid 7961) 7s; run: log: (pid 1167) 906s

So basically my graylog-server is down, I tried restart and reconfigure, but still the server is down. I hope you help me overcome this issue.

Cheers,

**Comment by mariussturm** [comment]
We are using GitHub issues for tracking bugs in Graylog itself, but this doesn't look like one. Please post this issue to our [discussion forum](https://community.graylog.org/) or join the [#graylog channel on freenode IRC](https://webchat.freenode.net/?channels=%23graylog).

Thank you!

**Comment by erunixgila** [comment]
![image](https://user-images.githubusercontent.com/13446633/50275769-c3d7e680-0472-11e9-942e-7b78784fb046.png)


![image](https://user-images.githubusercontent.com/13446633/50275813-de11c480-0472-11e9-991b-9608107ea5eb.png)
